src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -106,25 +107,10 @@ static bool operator==(const irep_idt &what, const patternt &pattern)
   return pattern==what;
 }
 
-const size_t SLOTS_PER_INTEGER(1u);
-const size_t INTEGER_WIDTH(64u);
-static size_t count_slots(
-  const size_t value,
-  const code_typet::parametert &param)
-{
-  const std::size_t width(param.type().get_unsigned_int(ID_width));
-  return value+SLOTS_PER_INTEGER+width/INTEGER_WIDTH;
-}
-
-static size_t get_variable_slots(const code_typet::parametert &param)
-{
-  return count_slots(0, param);
-}
-
 static irep_idt strip_java_namespace_prefix(const irep_idt to_strip)
 {
   const auto to_strip_str=id2string(to_strip);
-  assert(has_prefix(to_strip_str, "java::"));
+  PRECONDITION(has_prefix(to_strip_str, "java::"));
   return to_strip_str.substr(6, std::string::npos);
```

Comment:
@peterschrammel, I dind't change that code, only the assert -> precondition, as I was getting warnings by the linter. I can do the change that you suggest, if you want.